### PR TITLE
[RFC] Simplify post-deployment checks

### PR DESCRIPTION
**Pull request by tvararu**
## Context

Now that we have reliable automation for checking the health of our azure resources, and smoke tests that run post-deployment, we can significantly simplify our post-deployment checks.

## Changes proposed in this pull request

Rewrite our deployment guidelines, removing most manual checks.

## Guidance to review

Are we comfortable moving in this direction?

## Link to Trello card

None.

## Things to check

- [x] This code doesn't rely on migrations in the same Pull Request
- [x] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [x] API release notes have been updated if necessary
- [x] New environment variables have been [added to the Azure config](https://github.com/DFE-Digital/apply-for-teacher-training#azure-hosting-devops-pipeline)

### CODE REVIEW

**Comment by duncanjbrown** [comment]
This covers https://trello.com/c/o2hMRz9v/302-update-deployment-documentation-to-remove-azure-dashboard-instructions

File: docs/deployment.md [hunk 1]
```diff
@@ -41,23 +41,24 @@ Summarise what you're deploying and tell the team in Slack on the `#twd_apply` c
...
 
-## 6. Test on production
+## 4. Check if production and sandbox are deployed successfully
 
-Wait until the deploy finishes and run through the [post-deployment
-checklist](deployment_checklist.md) before declaring the deploy finished.
+Check the `#twd_apply_tech` channel in Slack for runtime errors from
+Sentry or the smoke tests.
 
 ## 7. Move deploy cards to done
```
Comment (duncanjbrown) [inline docs/deployment.md:63]: This is now step 5

Comment (tvararu) [reply docs/deployment.md:63]: So we move deploy cards to done after we deploy to staging?

Comment (duncanjbrown) [reply docs/deployment.md:63]: I can't read the markdown diff! Please ignore me 🙂

**Pull request merged**